src/test/java/ay2021s1_cs2103_w16_3/finesse/logic/parser/budget/SetSavingsGoalCommandParserTest.java

```diff
@@ -22,26 +25,21 @@ public void parse_emptyArg_throwsParseException() {
 
     @Test
     public void parse_invalidArgs_throwsParseException() {
-        assertParseFailure(parser, "a", String.format(MESSAGE_INVALID_COMMAND_FORMAT,
-                SetSavingsGoalCommand.MESSAGE_USAGE));
-        assertParseFailure(parser, "-5", String.format(MESSAGE_INVALID_COMMAND_FORMAT,
-                SetSavingsGoalCommand.MESSAGE_USAGE));
-        assertParseFailure(parser, "$", String.format(MESSAGE_INVALID_COMMAND_FORMAT,
-                SetSavingsGoalCommand.MESSAGE_USAGE));
-        assertParseFailure(parser, "123.4", String.format(MESSAGE_INVALID_COMMAND_FORMAT,
-                SetSavingsGoalCommand.MESSAGE_USAGE));
-        assertParseFailure(parser, "567.890", String.format(MESSAGE_INVALID_COMMAND_FORMAT,
-                SetSavingsGoalCommand.MESSAGE_USAGE));
-        assertParseFailure(parser, "2 hello", String.format(MESSAGE_INVALID_COMMAND_FORMAT,
-                SetSavingsGoalCommand.MESSAGE_USAGE));
+        // no whitespace in front
+        assertParseFailure(parser, PREFIX_AMOUNT + VALID_AMOUNT_INTERNSHIP,
+                String.format(MESSAGE_INVALID_COMMAND_FORMAT, SetSavingsGoalCommand.MESSAGE_USAGE));
+        // preamble before amount prefix
+        assertParseFailure(parser, "hello" + AMOUNT_DESC_INTERNSHIP,
```

Comment:
```suggestion
        assertParseFailure(parser, PREAMBLE_NON_EMPTY + AMOUNT_DESC_INTERNSHIP,
```

You can use `PREAMBLE_NON_EMPTY` which can be found in `CommandTestUtil`